src/main/java/carsharing/connection/PropertiesDataLoader.java

```diff
@@ -0,0 +1,32 @@
+package carsharing.connection;
+
+import carsharing.Constants;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Properties;
+
+public class PropertiesDataLoader {
+    public String getProperty(String propertyName) throws IOException {
+        String propertyValue;
+        Properties properties = new Properties();
+        try (FileInputStream inputStream = getFileFromResources(Constants.PROPERTIES)) {
```

Comment:
seems like you load the property file each time you invoke this method

Reply:
Done